task_12/src/main.scss

```diff
@@ -0,0 +1,216 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,700');
+@import "reset";
+@import "variables";
```

Comment:
`variables` must be first file you import because everything else depends on it.

Reply:
@viktor-yakubiv thanks again! I took into account your comments.
After completing the refactoring of my weather application I will work on
file structure of the pizza app.
Because at the moment I'm not sure how to properly structure it.